shell/platform/fuchsia/flutter/vsync_recorder.h

```diff
@@ -30,7 +30,7 @@ class VsyncRecorder {
   // to be called in |scenic::Session::Present2| immedaite callbacks with the
   // presentation info provided by Scenic.  Only the next vsync
   // information will be saved (in order to handle edge cases involving
-  // multiple Scenic sessions in the same process).  This function is safe to
+  // multiple Scenic sessions in the same process). This function is safe to
   // call from any thread.
```

Comment:
If you follow my other suggestion below, then this class no longer has a reason to exist.  Everything happens on one thread now, and mostly inside of the SessionConnection.

At that point, you can just collapse this class into SessionConnection without violating any sort of encapsulation.

Reply:
This also is not possible now unfortunately, so I filed  flutter/flutter#81984